core/kazoo_convert/doc/fax_converter.md

```diff
@@ -0,0 +1,175 @@
+# Kazoo Fax Converter
+
+This module provide a consistant core interface for fax file format conversions. The module is enabled as the default fax converter, but this can be easily extended by adding other modules to the converter.
+
+## Fax Converter Installation Considerations
+When using the default converter module several system dependencies were introduced. For default conversions from PDF to TIFF, The system packages `ghostscript` must be installed, for default conversion from tiff to pdf, `libtiff-tools` must be installed. If conversion to openxml formatted documents (docx, doc, xlsx, xls) is enabled, the dependency `libreoffice` must be installed. If a dependency is not installed, the converter command will simply fail on the system and the fax converter will return an error. If the converters commands are substituted this.
```

Comment:
Instead of mentioning requirement in paragraph, add a new Requirement section and list all commands and what they're use for.